### ISSUE TITLE
[BUG] pull-secret  shown in clear text within the logs

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/crc#3856]
Running a restart scenario for `microshift` (setup -> start -> stop ->start) on the 2nd start checking if pull secret is already present the result of the check is shown on the logs... so the pull-secret is exposed in clear text:

```
time="2023-10-03T16:54:11+02:00" level=info msg="Starting Microshift service... [takes around 1min]"
time="2023-10-03T16:54:11+02:00" level=debug msg="Using root access: Checking if pull secret already present in the VM"
time="2023-10-03T16:54:11+02:00" level=debug msg="Running SSH command: sudo cat /etc/crio/openshift-pull-secret"
time="2023-10-03T16:54:12+02:00" level=debug msg="SSH command results: err: <nil>, output: {\"auths\":{\"cloud.o....
```

**Comment by adrianriobo** [comment]
Most probably it affects all platforms but need to ensure, will update tags accordingly

**Comment by adrianriobo** [comment]
Fixed by #3858
